roottest/scripts/build.C

```diff
@@ -75,9 +75,6 @@ void build(const char *filename, const char *lib = 0, const char *obj = 0, const
    }
    // fprintf(stderr,"from %s to %s\n",filename,fname.Data());
    int result = gSystem->CompileMacro(fname,"kc");
-#elif defined(CMakeEnvironment) && defined(CMakeBuildDir)
-   // fprintf(stderr,"CmakeBuildDir: %s filename: %s", CMakeBuildDir, filename);
-   int result = gSystem->CompileMacro(filename,"kc-", libname, CMakeBuildDir);
```

Comment:
This seems to be an unintentional behavior change.  If I read correctly, before this PR, `ROOTTEST_COMPILE_MACRO` was always setting `CMakeEnvironment` and `CMakeBuildDir` and thus always passing ` "kc-" `, while it now passes "only" ` "kc" ` thus removing the `-` and changing the output directory layout.